README.md

```diff
@@ -1,4 +1,22 @@
 displayer-multiselect-suggest
 =============================
 
+##Description
 A custom suggest displayer for DBList object properties which adapts the suggest feature to multiple selection.
+
+##Usage
+* To activate it, you must edit the XClass and set, for the target property, the fields Use suggest and Multiple select to true, and the field Custom display to:
+{{include document="XWiki.SuggestDisplay" /}}
+
+* To activate the itemsEditable mode that allows you to create new items and edit selected items, keep the configuration above and precede the {{include}} macro with below code.
+
+```
+{{velocity}}
+#set($SuggestDisplay_itemsEditable = true)
```

Comment:
I like the fact that you have introduced this variable, but the name is inconsistent with other variables: `$isUser, $isLink, $isExternalLink, $displayAddButton`...